localisation/atw_replace_l_english.yml

```diff
@@ -56,9 +56,6 @@
  despotism_desc:0 "Despotism is a form of government in which a single person rules with absolute power."
  oligarchism:0 "Oligarchic"
  oligarchism_desc:0 "Oligarchy is a form of government in which a group of people rule with absolute power."
- moderatism:1 "Moderatism"
- moderatism_desc:1 "Moderatism is a form of government which avoids the extremes of left and right by taking a moderate position or course of action."
- centrism:1 "Centrism"
```

Comment:
You remove a used ideology here.

Name however now is `neu_centrism` or `dem_centrism`

Reply:
see discussion above on moderatism